tests/search_tests.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from dsalgo.Search import Search
+array = [2, 4, 6, 8, 10, 1]
+
```

Comment:
write proper test cases

Reply:
I can cover all corner cases but you must decide upon how you want to design it. 
Could you update the details of your expected design here and in the issue section so that I can refer to that and meet your expectations.

Reply:
for writing code you can view the linked list implementation.
You don't need to follow a proper template only thing that we expect you from is modular code.
Regarding test cases you have to think of all edge cases by your own.
Designing is your part but be concise of what you are doing.